css/style.css

```diff
@@ -0,0 +1,662 @@
+:root {
+  --white: white;
+}
+
+/* CSS Reset */
+* {
+  margin: 0;
+  padding: 0;
+  max-width: 1440px;
+  box-sizing: border-box;
+}
+
+/* Element Type Selectors */
+body {
+  font-family: 'Comic Neue', cursive;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* Images */
+.menu-cat img {
+  width: 30px;
+  height: 30px;
+  margin-left: 5px;
+}
+
+.img-transition img {
+  transform: rotate(-15deg);
+  cursor: pointer;
+}
+
+.signup-articles img {
+  width: 150px;
+  height: 40%;
+  padding-top: 50px;
+  padding-bottom: 30px;
+  border-radius: 50%;
+}
+
+.upcoming-section img {
+  width: 150px;
+  padding-top: 50px;
+  padding-bottom: 30px;
+  border-radius: 50%;
+}
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS. For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)

You are repeating:
```
  width: 150px;
  padding-top: 50px;
  padding-bottom: 30px;
  border-radius: 50%;
  border: 3px solid gray;
  background: #a9a9a9;
  display: flex;
  display: grid;
  position: absolute;
  font-weight: bold;
  justify-content: center;
  text-align: center;
  color: var(--white);
```

  👉 **_Tip:_**Here you have an explanation of how you can implement it:

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
  and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  
  ...
  <div class="class2 common-class">  
  ...
  <footer class="class3 common-class">   
  ...
  ```